addons/l10n_ro_edi_stock/models/stock_picking.py

```diff
@@ -0,0 +1,864 @@
+    'DJ': '16',
+    'GL': '17',
+    'GJ': '18',
+    'HR': '19',
+    'HD': '20',
+    'IL': '21',
+    'IS': '22',
+    'IF': '23',
+    'MM': '24',
+    'MH': '25',
+    'MS': '26',
+    'NT': '27',
+    'OT': '28',
+    'PH': '29',
+    'SM': '30',
+    'SJ': '31',
+    'SB': '32',
+    'SV': '33',
+    'TR': '34',
+    'TM': '35',
+    'TL': '36',
+    'VS': '37',
+    'VL': '38',
+    'VN': '39',
+    'B': '40',
+    'CL': '51',
+    'GR': '52',
+}
+
+
+class Picking(models.Model):
+    _inherit = 'stock.picking'
+
+    # Document fields
+    l10n_ro_edi_stock_document_ids = fields.One2many(comodel_name='l10n_ro_edi.document', inverse_name='picking_id')
+    l10n_ro_edi_stock_document_message = fields.Char(compute='_compute_l10n_ro_edi_stock_current_document_data')
+    l10n_ro_edi_stock_document_load_id = fields.Char(compute='_compute_l10n_ro_edi_stock_current_document_data', string="eTransport Id")
+    l10n_ro_edi_stock_document_uit = fields.Char(compute='_compute_l10n_ro_edi_stock_current_document_data', string="eTransport UIT")
+    l10n_ro_edi_stock_state = fields.Selection(selection=DOCUMENT_STATES, compute='_compute_l10n_ro_edi_stock_current_document_data', string="eTransport Status")
+
+    # Data fields
+    l10n_ro_edi_stock_operation_type = fields.Selection(selection=OPERATION_TYPES, string="Operation Type")
+    l10n_ro_edi_stock_available_operation_scopes = fields.Char(compute='_compute_l10n_ro_edi_stock_available_operation_scopes')
+    l10n_ro_edi_stock_operation_scope = fields.Selection(selection=OPERATION_SCOPES, string="Operation Scope")
+
+    l10n_ro_edi_stock_vehicle_number = fields.Char(string="Vehicle Number", size=20)
+    l10n_ro_edi_stock_trailer_1_number = fields.Char(string="Trailer 1 Number", size=20)
+    l10n_ro_edi_stock_trailer_2_number = fields.Char(string="Trailer 2 Number", size=20)
+
+    l10n_ro_edi_stock_available_start_loc_types = fields.Char(compute='_compute_l10n_ro_edi_stock_available_location_types')
+    l10n_ro_edi_stock_start_loc_type = fields.Selection(selection=LOCATION_TYPES, string="Location Type", default='location')
+
+    l10n_ro_edi_stock_available_end_loc_types = fields.Char(compute='_compute_l10n_ro_edi_stock_available_location_types')
+    l10n_ro_edi_stock_end_loc_type = fields.Selection(selection=LOCATION_TYPES, string="Location Type", default='location')
+
+    # Data fields for every location type
+    l10n_ro_edi_stock_start_bcp = fields.Selection(selection=BORDER_CROSSING_POINTS, string="Border Crossing Point")
+    l10n_ro_edi_stock_start_customs_office = fields.Selection(selection=CUSTOMS_OFFICES, string="Custom Office")
+    l10n_ro_edi_stock_start_state_id = fields.Many2one(comodel_name='res.country.state', string="State", domain="[('country_id.code', '=', 'RO')]")
+    l10n_ro_edi_stock_start_city = fields.Char(string="City")
+    l10n_ro_edi_stock_start_street = fields.Char(string="Street")
+    l10n_ro_edi_stock_start_zip = fields.Char(string="Postal Code")
```

Comment:
We already have the address of the warehouse and the destination address. Are those fields really necessary?